python/.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length = 120
+extend-ignore = E203, W503, E501, E711, E712, N803
```

Comment:
It doesn't look like we break any of these rules, why not enforce them? except for maybe `W503` which is deprecated

Reply:
This is the recommended list from `black`. It's the list of rules where flake8 is not compliant and is incompatible with black. That's why we want to ignore them.